system-container/config.json.template

```diff
@@ -0,0 +1,221 @@
+				"hard": 1024,
+				"soft": 1024
+			}
+		],
+		"noNewPrivileges": true
+	},
+	"root": {
+		"path": "rootfs",
+		"readonly": true
+	},
+	"mounts": [
+		{
+			"destination": "/proc",
+			"type": "proc",
+			"source": "proc"
+		},
+		{
+			"destination": "/dev",
+			"type": "tmpfs",
+			"source": "tmpfs",
+			"options": [
+				"nosuid",
+				"strictatime",
+				"mode=755",
+				"size=65536k"
+			]
+		},
+		{
+			"destination": "/dev/pts",
+			"type": "devpts",
+			"source": "devpts",
+			"options": [
+				"nosuid",
+				"noexec",
+				"newinstance",
+				"ptmxmode=0666",
+				"mode=0620",
+				"gid=5"
+			]
+		},
+		{
+			"destination": "/dev/shm",
+			"type": "tmpfs",
+			"source": "shm",
+			"options": [
+				"nosuid",
+				"noexec",
+				"nodev",
+				"mode=1777",
+				"size=65536k"
+			]
+		},
+		{
+			"destination": "/dev/mqueue",
+			"type": "mqueue",
+			"source": "mqueue",
+			"options": [
+				"nosuid",
+				"noexec",
+				"nodev"
+			]
+		},
+		{
+			"destination": "/sys",
+			"type": "sysfs",
+			"source": "sysfs",
+			"options": [
+				"nosuid",
+				"noexec",
+				"nodev",
+				"ro"
+			]
+		},
+		{
+		        "type": "bind",
+                        "source": "$SSH_ROOT",
+                        "destination": "/opt/app-root/src/.ssh",
+                        "options": [
+                                 "bind",
+		                 "rw",
+		                 "mode=755"
+	                ]
+                },
+		{
+		        "type": "bind",
+                        "source": "$SSH_ROOT",
+                        "destination": "/root/.ssh",
+                        "options": [
+                                 "bind",
+		                 "rw",
+		                 "mode=755"
+	                ]
+                },
+		{
+		        "type": "bind",
+                        "source": "$VAR_LIB_OPENSHIFT_INSTALLER",
+                        "destination": "/var/lib/openshift-installer",
+                        "options": [
+                                 "bind",
+		                 "rw",
+		                 "mode=755"
+	                ]
+                },
+		{
+		        "type": "bind",
+                        "source": "$VAR_LOG_OPENSHIFT_LOG",
+                        "destination": "/var/log/ansible.log",
+                        "options": [
+                                 "bind",
+		                 "rw",
+		                 "mode=755"
+	                ]
+                },
+		{
+			"destination": "/root/.ansible",
+			"type": "tmpfs",
+			"source": "tmpfs",
+			"options": [
+				"nosuid",
+				"strictatime",
+				"mode=755"
+			]
+		},
+		{
+			"destination": "/tmp",
+			"type": "tmpfs",
+			"source": "tmpfs",
+			"options": [
+				"nosuid",
+				"strictatime",
+				"mode=755"
+			]
+		},
+		{
+		        "type": "bind",
+                        "source": "$INVENTORY_FILE",
+                        "destination": "/etc/ansible/hosts",
+                        "options": [
+                                 "bind",
```

Comment:
Looks like there's an extra space messing up the indentation here.

Here and a handful of other occurrences above inside of `"options"` blocks.